CHANGELOG.md

```diff
@@ -6,6 +6,24 @@ All notable changes to **NCronJob** will be documented in this file. The project
 
 ## [Unreleased]
 
+## [2.1.0] - 2024-04-23
+
+### Added
+- **Retry Mechanism:** Improved robustness of job executions, especially in cases of transient failures. Includes exponential backoff and fixed interval strategies.
```

Comment:
Please also add your Github handle - you deserve the credit.
(See for example version 2.0.5)